src/app/searchResults/searchResults.component.js

```diff
@@ -45,6 +45,9 @@ class SearchResultsController {
           this.searchResults = results;
         }
         this.loadingResults = false;
+      }, () => {
+        this.searchResults = [];
```

Comment:
Should we show the user an error message that maybe includes a message to try again?

Also would it be useful to log this to rollbar?